src/main/scala/firrtl/passes/LowerTypes.scala

```diff
@@ -0,0 +1,274 @@
+/*
```

Comment:
In future, it would be better to not move the pass to a separate file in the same commit as you modify it -- it's hard to see what changed.  (I guess in this case, it was a near-total rewrite, so it doesn't matter much anyway.)